Doc/library/functions.rst

```diff
@@ -1979,7 +1979,7 @@ are always available.  They are listed here in alphabetical order.
 
    .. index::
       statement: import
-      module: imp
+      pair: module; builtins
```

Comment:
This was the merge conflict -- I believe that ``__import__`` is more strictly part of ``builtins`` in 3.11, hence going with the above rather than ``pair: module; imp``.

A